src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+public class Rectangle extends Quadrangle {
+    public Rectangle(String color, int width, int height) {
+        super(color, width, height);
+    }
+
+    @Override
+    public double getArea() {
+        return super.getArea();
+    }
+
+    @Override
+    public String draw() {
+        return "Figure: rectangle ," + super.draw();
```

Comment:
The `draw` method should not directly concatenate strings to form the message. Consider using `String.format` for better readability. Also, the comma after 'rectangle' should be followed by a space.